api/modules/crudRefactorClean.js

```diff
@@ -0,0 +1,316 @@
+                return courses
+            })
+        )
+    }
+
+    //TOPICS
+    const fetchTopicsById = (params) => {
+        const { topicId } = params 
+        return (
+            Topic.findAll({
+                where: {
+                    id: topicId
+                }
+            })
+            .then(topics => {
+                console.log('All Topics:', JSON.stringify(topics, null, 4));
+                return topics
+            })
+        )
+    }
+    const fetchTopics = (params) => {
+        return (
+            Topic.findAll({
+            })
+            .then(topics => {
+                console.log('All Topics:', JSON.stringify(topics, null, 4));
+                return topics
+            })
+        )
+    }
+    const fetchTopicsByCourse = (params) => {
+        const { courseId } = params 
+        return (
+            Topic.findAll({
+                where: {
+                    course_id: courseId
+                }
+            })
+            .then(topics => {
+                console.log('All Topics by Course:', courseId, JSON.stringify(topics, null, 4));
+                return topics
+            })
+        )
+    }
+    const createTopics = (params) => {
+        const {
+            title,
+            order,
+            rich_text_content,
+            created_on,
+            course_id,
+        } = params.values;
+
+        Topic.create(
+            {
+                title,
+                order,
+                rich_text_content,
+                created_on,
+                course_id,
+            }
+        )
+        .then(topics => {
+            console.log('Id of new topic:', topics.id);
+        });
+
+    } 
+    const deleteTopics = (params) => {
+        const { topicId } = params 
+        return (
+            Topic.destroy({
+                where: {
+                    id: topicId
+                }
+            })
+            .then(topics => {
+                console.log('Course deleted correctly');
+                return topics
+            })
+        )
+    }
+    const fetchTopicsByLanguage = (params) => {
+        const { languageId } = params 
+        return (
+            Topic.findAll({
+                where: {
+                    language_id: languageId
+                }
+            })
+            .then(topics => {
+                console.log('Topics:', JSON.stringify(courses, null, 4));
+                return topics
+            })
+        )
+    }
+    //USERS
+    const fetchUsers = (params) => {
+        return (
+            User.findAll({
+            })
+            .then(users => {
+                console.log('All Topics:', JSON.stringify(users, null, 4));
+                return users
+            })
+        )
+    }
+    const createUsers = (params) => {
+        const {
+            email,
+            hashed_password,
+            date_registered,
+            google_id,
+        } = params.values;
+
+        User.create(
```

Comment:
wrap in a return statement (this will be important when trying to call this function from another module)